qcodes/instrument_drivers/AlazarTech/ATS.py

```diff
@@ -408,9 +459,10 @@ def config(self, clock_source=None, sample_rate=None, clock_edge=None,
                            self.parameters['coupling' + str(i)],
                            self.parameters['channel_range' + str(i)],
                            self.parameters['impedance' + str(i)])
-            self._call_dll('AlazarSetBWLimit',
-                           self._handle, i,
-                           self.parameters['bwlimit' + str(i)])
+            if bwlimit is not None:
+                self._call_dll('AlazarSetBWLimit',
+                               self._handle, i,
```

Comment:
not sure if handle should be binary here as well, the ATS9440 doesn't have bw limit